### ISSUE TITLE
is there any version of logtrail for kibana 5.6.10

### ISSUE THREAD

**Issue by nuaays** [sivasamyk/logtrail#299]
I was using kibana 5.6.10, and install logstrail for 5.6.5, and got this warning 

```
[08:54:20.304] [warning] Plugin "logtrail" was disabled because it expected Kibana version "5.6.5", and found "5.6.10".

```

**Comment by sivasamyk** [comment]
No. You can follow the instructions here to update the kibana version to 5.6.10. https://github.com/sivasamyk/logtrail/blob/master/docs/how_to.md#2-update-kibanaversion-in-logtrail-plugin-archive

**Comment by choffee** [comment]
A release for the 5.6.13 security update would be really handy.
